src/elements/Promise.tsx

```diff
@@ -0,0 +1,18 @@
+/* @jsx MD */
+import MD, { Component, PropsWithChildren } from "..";
+import { MarkdownPromise } from "../model";
+
+/**
+ * Renders a single promise that resolves to a group of tags to markdown asynchronously.
+ * Throws an error if multiple children is provided.
+ * @example
+ *   ```js
+ *   render(<Promise>{new Promise((res) => res("Hello"))}</Promise>)
+ *   ===
+ *   `Hello`
+ *   ```
+ */
+export const MdPromise: Component<PropsWithChildren<
+  unknown,
+  MarkdownPromise
+>> = ({ children }) => <mdPromise>{children}</mdPromise>;
```

Comment:
I'm not sure why test coverage for this line is missing. I can have a look at that specifically end of week :)